MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,83 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    private class TestClipRequest {
+        let start: Float
+        let end: Float
+
+        init(start: Float, end: Float) {
+            self.start = start
+            self.end = end
+        }
+    }
+
+    private class RecordDelegate: RecordCoordinatorDelegate {
+        private let onRecordingStopped: () -> Void
+
+        init(onRecordingStopped: @escaping () -> Void) {
+            self.onRecordingStopped = onRecordingStopped
+        }
+
+        func recordingStarted(path: String) {
+        }
+
+        func recordingStopped(recordingPath: RecordingPath) {
+            onRecordingStopped()
```

Comment:
`recordingStopped` method supplies a recording path that may be used for checking the correctness of its contents. Please use it instead of checking `DocumentsLocation.currentRecording.path`.

Reply:
done